### PR TITLE
Bug 952456 - Add "Copy Image" option

### PR DESCRIPTION
**Pull request by BorisChiou**
Provide "copy image" option on context menu

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3c4a7380ba927d764c2db21e2087a09806f79735

File: apps/system/js/browser_context_menu.js [hunk 1]
```diff
@@ -73,13 +73,22 @@
...
-            icon: choice.icon,
-            callback: function() {
-              detail.contextMenuItemSelected(choice.id);
-            }
-          });
+          var itemObj = null;
+          if (choice.id == 'copy-image') {
+            itemObj = {
+              id: choice.id,
+              label: _(choice.id),
```
Comment (timdream) [inline apps/system/js/browser_context_menu.js]: What makes `copy-image` special? If it's just about missing graphics, simply commit a temporary one is better than mess with the code like this....

File: apps/system/locales/system.en-US.properties [hunk 2]
```diff
@@ -600,6 +600,7 @@ save-link=Save linked file
 share-image=Share Image
 share-video=Share Video
 share-audio=Share Audio
+copy-image=Copy Image
```
Comment (timdream) [inline apps/system/locales/system.en-US.properties]: Agree we need a UI label regardless.

File: apps/system/js/browser_context_menu.js (hunk 1 above)
Comment (timdream) [inline apps/system/js/browser_context_menu.js]: Never mind, I've figure out why it's needed here...

File: apps/system/js/browser_context_menu.js [hunk 3]
```diff
@@ -73,13 +73,22 @@
...
         detail.contextmenu.items.forEach(function(choice, index) {
-          items.push({
-            label: choice.label,
-            icon: choice.icon,
-            callback: function() {
-              detail.contextMenuItemSelected(choice.id);
-            }
-          });
+          var itemObj = null;
+          if (choice.id == 'copy-image') {
```
Comment (timdream) [inline apps/system/js/browser_context_menu.js]: `switch ... case` is better than `if .. else` here IMHO.

File: apps/system/js/browser_context_menu.js (hunk 1 above)
Comment (BorisChiou) [inline apps/system/js/browser_context_menu.js]: OK, I could use a temporary one for copy-image case. Thanks.

File: apps/system/js/browser_context_menu.js (hunk 3 above)
Comment (BorisChiou) [inline apps/system/js/browser_context_menu.js]: Sure, I will upload new patch to use switch case.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8034f561e5500471cc764c1fef7c286cfef98d65

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31fbbc090a8f57c277999dd6cd3c659396e4cc2d

**Comment by BorisChiou** [comment]
Fix 1 jshint error.

**Pull request merged**